.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -1,18 +1,32 @@
+---
 name: Report an issue
-description: Tell us about something that's not working the way we (probably) intend
+description:
+  Tell us about something that is not working the way we (probably) intend.
 body:
-  - type: input
+  - type: markdown
+    attributes:
+      value: >
+        Thank you for contributing to Elixir! :heart:
+
+        Please, do not use this form for guidance, questions or support.
+        Try instead in [Elixir Forum](https://elixirforum.com),
+        the [IRC Chat](https://web.libera.chat/#elixir),
+        [Stack Overflow](https://stackoverflow.com/questions/tagged/elixir),
+        [Slack](https://elixir-slackin.herokuapp.com),
+        [Discord](https://discord.gg/elixir) or in other online communities.
```

Comment:
Can we add these links as well? I didn't add them because I wasn't sure if it was OK to promote them

Reply:
I took them from elixir-lang.org sidebar so considered them sort of official.

Reply:
I added them just in case 🤷🏻

Reply:
Ohh. I see what you mean now in that new PR. Let's see.